packages/core/src/app/pools/[poolAddress]/page.tsx

```diff
@@ -556,12 +590,14 @@ export default function SwapPage({
                 </Box>
                 <PoolLiquidityForm
                   {...provideLiquidityProps}
+                  addLiquidityButtonDisabled={addLiquidityButtonDisabled}
```

Comment:
Just put your logic here, like `addLiquidityButtonDisabled={!(appStore.wallet.address && tokenAValue !== "" && tokenBValue !== "")}`

Be sure to check if the input exceeds the balance.